src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/MicrosoftNetFrameworkAnalyzersResources.resx

```diff
@@ -139,91 +139,91 @@
     <value>Set locale for data types</value>
   </data>
   <data name="SetLocaleForDataTypesDescription" xml:space="preserve">
-    <value>The locale determines culture-specific presentation elements for data, such as formatting that is used for numeric values, currency symbols, and sort order. When you create a DataTable or DataSet, you should explicitly set the locale.</value>
+    <value>The locale determines culture-specific presentation elements for data, such as formatting that is used for numeric values, currency symbols, and sort order. When you create a 'DataTable' or 'DataSet', you should explicitly set the locale.</value>
   </data>
   <data name="SetLocaleForDataTypesMessage" xml:space="preserve">
     <value>Set locale for data types</value>
   </data>
   <data name="DoNotMarkServicedComponentsWithWebMethodTitle" xml:space="preserve">
-    <value>Do not mark serviced components with WebMethod</value>
+    <value>Do not mark serviced components with 'WebMethod'</value>
```

Comment:
I know @sharwell was in favor of using the `[WebMethod]` syntax but if we want to be language agnostic it feels weird.